packages/workflow/tsconfig.json

```diff
@@ -1,12 +1,10 @@
 {
-	"extends": "@n8n/typescript-config/tsconfig.common.json",
+	"extends": "@n8n/typescript-config/modern/tsconfig.json",
 	"compilerOptions": {
 		"rootDir": ".",
-		"baseUrl": "src",
-		"paths": {
-			"@/*": ["./*"]
```

Comment:
Why are we dropping the `@/*` path here?

Reply:
To avoid the need to run `tsconfig-paths` after every build twice (esm and cjs).